### PR TITLE
Ci

### PR DESCRIPTION
**Pull request by tavisca-vjola**
@Ashwarya-Tavisca

Added both ci and cd in single jenkins file

### CODE REVIEW

**Comment by tavisca-vjola** [comment]
@tavisca-tgoyal 
please review my code

File: Dockerfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
 FROM microsoft/dotnet:2.2-aspnetcore-runtime-nanoserver-1803 AS base
 COPY ./webapi/bin/Debug/netcoreapp2.2/publish .
 ENTRYPOINT ["dotnet", "webapi.dll"]
+#ARG MYFILENAME="Default"
+#ENTRYPOINT ["dotnet", "${MYFILENAME}.dll"]
```
Comment (tavisca-tgoyal) [inline Dockerfile:5]: I am tavisca t goyal bayi

**Review by tavisca-tgoyal — CHANGES_REQUESTED** [review]

**Comment by tavisca-vjola** [comment]
please review my code
@tavisca-tgoyal

**Comment by tavisca-vjola** [comment]
please review my code @tavisca-tgoyal

File: Jenkinsfile [hunk 2]
```diff
@@ -8,50 +8,40 @@ pipeline {
      string(name: 'GIT_REPO_PATH', defaultValue: 'https://github.com/tavisca-vjola/web_api.git')
      string(name:'IMAGE_NAME',defaultValue:'sample-webapi',description: 'Enter the image name')
       string(name: 'DOCKER_LOGIN', defaultValue: 'vjmsai', description: 'Enter Login')
-       string(name: 'DOCKER_PASSWORD', defaultValue: 'Jola@1998', description: 'Enter Password')
+       password(name: 'DOCKER_PASSWORD', defaultValue: 'Jola@1998', description: 'Enter Password')
```
Comment (tavisca-tgoyal) [inline Jenkinsfile:11]: please dont  use it buddy

**Review by tavisca-tgoyal — CHANGES_REQUESTED** [review]

**Comment by tavisca-vjola** [comment]
please add this file @tavisca-tgoyal

**Comment by tavisca-vjola** [comment]
please review@tavisca-spanda

**Comment by tavisca-vjola** [comment]
please review @tavisca-spanda

File: Dockerfile [hunk 3]
```diff
@@ -1,3 +1,5 @@
 FROM microsoft/dotnet:2.2-aspnetcore-runtime-nanoserver-1803 AS base
```
Comment (tavisca-spanda) [inline Dockerfile:1]: I saw

**Review by tavisca-spanda — COMMENTED** [review]
Here it is

File: Jenkinsfile (hunk 2 above)
Comment (tavisca-spanda) [inline Jenkinsfile:11]: Don't do this.

**Review by tavisca-spanda — COMMENTED** [review]
See the comment

**Comment by tavisca-vjola** [comment]
Please comment please

**Comment by tavisca-vjola** [comment]
comment @tavisca-spanda

File: Dockerfile (hunk 1 above)
Comment (tavisca-spanda) [inline Dockerfile:5]: Dekh lo

File: Jenkinsfile [hunk 4]
```diff
@@ -8,50 +8,40 @@ pipeline {
      string(name: 'GIT_REPO_PATH', defaultValue: 'https://github.com/tavisca-vjola/web_api.git')
      string(name:'IMAGE_NAME',defaultValue:'sample-webapi',description: 'Enter the image name')
       string(name: 'DOCKER_LOGIN', defaultValue: 'vjmsai', description: 'Enter Login')
-       string(name: 'DOCKER_PASSWORD', defaultValue: 'Jola@1998', description: 'Enter Password')
```
Comment (tavisca-spanda) [inline Jenkinsfile:11]: I concur

**Review by tavisca-spanda — COMMENTED** [review]
check it

**Pull request closed without merging**